dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/interruptedjobwindow/JobInfoPresenter.java

```diff
@@ -88,31 +91,34 @@
     private final DeepStorageBrowserPresenter deepStorageBrowserPresenter;
     private final DateTimeUtils dateTimeUtils;
     private final SettingsStore settingsStore;
-    private final JobTaskElement jobTaskElement;
+    private final RecoverJobFactory recoverJobFactory;
     private final SavedJobPrioritiesStore savedJobPrioritiesStore;
+    private final JobTaskElement.JobTaskElementFactory jobTaskElementFactory;
     private Stage stage;
 
     @Inject
     public JobInfoPresenter(final ResourceBundle resourceBundle,
-                            final Ds3Common ds3Common,
-                            final Workers workers,
-                            final JobWorkers jobWorkers,
-                            final JobInterruptionStore jobInterruptionStore,
-                            final ButtonCell.ButtonCellFactory buttonCellFactory,
-                            final LoggingService loggingService,
-                            final DateTimeUtils dateTimeUtils,
-                            final SettingsStore settingsStore,
-                            final JobTaskElement jobTaskElement,
-                            final SavedJobPrioritiesStore savedJobPrioritiesStore,
-                            final DeepStorageBrowserPresenter deepStorageBrowserPresenter) {
+            final Ds3Common ds3Common,
+            final Workers workers,
+            final JobWorkers jobWorkers,
+            final JobInterruptionStore jobInterruptionStore,
+            final ButtonCell.ButtonCellFactory buttonCellFactory,
+            final LoggingService loggingService,
+            final DateTimeUtils dateTimeUtils,
```

Comment:
Does DateTimeUtils need to be injected?  This seems like it should be a static class.

Reply:
In the future DateTimeUtils could be injected to get the date in a localized format, for now it looks kind of funny.

Reply:
I guess my question is, why is a Util class being injected?  I thought all our Util classes only had static methods.

Reply:
Changing to static methods.